quantum/keymap_extras/keymap_french_afnor.h

```diff
@@ -0,0 +1,253 @@
+#define FR_LBRC ALGR(FR_LPRN) // [
+#define FR_RBRC ALGR(FR_RPRN) // ]
+#define FR_MACR ALGR(FR_LSQU) // ¯ (dead)
+#define FR_UNDS ALGR(FR_RSQU) // _
+#define FR_LDQU ALGR(FR_LDAQ) // “
+#define FR_RDQU ALGR(FR_RDAQ) // ”
+#define FR_DEG  ALGR(FR_QUOT) // °
+#define FR_CARN ALGR(FR_DCIR) // ˇ (dead)
+// Row 2
+#define FR_AE   ALGR(FR_A)    // æ
+#define FR_PND  ALGR(FR_Z)    // £
+#define FR_EURO ALGR(FR_E)    // €
+#define FR_REGD ALGR(FR_R)    // ®
+#define FR_LCBR ALGR(FR_T)    // {
+#define FR_RCBR ALGR(FR_Y)    // }
+#define FR_UGRV ALGR(FR_U)    // ù
+#define FR_DOTA ALGR(FR_I)    // ˙ (dead)
+#define FR_OE   ALGR(FR_O)    // œ
+#define FR_PERC ALGR(FR_P)    // %
+#define FR_MMNS ALGR(FR_MINS) // −
+#define FR_DAGG ALGR(FR_PLUS) // †
+// Row 3
+#define FR_THET ALGR(FR_Q)    // θ
+#define FR_SS   ALGR(FR_S)    // ß
+#define FR_DLR  ALGR(FR_D)    // $
+#define FR_CURR ALGR(FR_F)    // ¤ (dead monetary key)
+#define FR_DGRK ALGR(FR_G)    // µ (dead Greek key)
+#define FR_EU   ALGR(FR_H)    // Eu (dead European symbol key)
+#define FR_DSLS ALGR(FR_K)    // ∕ (dead)
+#define FR_PIPE ALGR(FR_L)    // |
+#define FR_INFN ALGR(FR_M)    // ∞
+#define FR_DIV  ALGR(FR_SLSH) // ÷
+#define FR_MUL  ALGR(FR_ASTR) // ×
+// Row 4
+#define FR_LEQL ALGR(FR_LABK) // ≤
+#define FR_EZH  ALGR(FR_W)    // ʒ
+#define FR_COPY ALGR(FR_X)    // ©
+#define FR_CCED ALGR(FR_C)    // ç
+#define FR_CEDL ALGR(FR_V)    // ¸ (dead)
+#define FR_DMNS ALGR(FR_B)    // − (dead)
+#define FR_DTIL ALGR(FR_N)    // ~ (dead)
+#define FR_IQUE ALGR(FR_DOT)  // ¿
+#define FR_IEXL ALGR(FR_COMM) // ¡
+#define FR_MDDT ALGR(FR_COLN) // ·
+#define FR_AEQL ALGR(FR_SCLN) // ≃
+
+/* Shift+AltGr symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │  ̑ │   │   │   │   │ ˝ │  ̏ │   │ — │ ‹ │ › │ ˚ │   │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │   │   │   │   │ ™ │   │   │  ̣ │   │ ‰ │ ‑ │ ‡ │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │   │   │   │   │   │ ˍ │   │   │   │   │ √ │ ¼ │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+ * │    │ ≥ │   │   │   │ ˛ │   │   │   │  ̦ │   │ ≠ │          │
+ * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+ * │    │    │    │                        │    │    │    │    │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ */
+// Row 1
+#define FR_IBRV S(ALGR(FR_AT))   //  ̑ (dead)
+#define FR_DACU S(ALGR(FR_LPRN)) // ˝ (dead)
+#define FR_DGRV S(ALGR(FR_RPRN)) //  ̏ (dead)
+#define FR_MDSH S(ALGR(FR_RSQU)) // —
+#define FR_LAQU S(ALGR(FR_LDAQ)) // ‹
+#define FR_RAQU S(ALGR(FR_RDAQ)) // ›
+#define FR_RNGA S(ALGR(FR_QUOT)) // ˚ (dead)
+// Row 2
+#define FR_TM   S(ALGR(FR_T))    // ™
+#define FR_DOTB S(ALGR(FR_I))    //  ̣ (dead)
+#define FR_PERM S(ALGR(FR_P))    // ‰
+#define FR_IDSH S(ALGR(FR_MINS)) // ‑
+#define FR_DDAG S(ALGR(FR_PLUS)) // ‡
+// Row 3
+#define FR_SMAC S(ALGR(FR_H))    // ˍ (dead)
```

Comment:
```suggestion
#define FR_MACB S(ALGR(FR_H))    // ˍ (dead)
```

Reply:
Let's be honest. I don't understand all the naming sometimes.
Why MACB (Second Macron?) instead of DMAC (Dead Macron) or SMAC (subscript Macron, which is the more accurate to me)?

Reply:
I'm deriving them (as much as possible) from the Unicode names. `MACB` = Macron Below
https://www.fileformat.info/info/unicode/char/0331/index.htm

Reply:
OK thanks for the info :)